### ISSUE TITLE
Unavailable erddap fails monitoring

### ISSUE THREAD

**Issue by gmaze** [euroargodev/argopy-status#1]
I get this error raised in the "Send ERDDAP status by email" step of the Actions "check-and-report":
```bash
Error: Input required and not supplied: subject
```

This originates from this error, the erddap argopy source cannot be loaded:
```python
/opt/hostedtoolcache/Python/3.6.12/x64/lib/python3.6/site-packages/argopy/utilities.py:84: UserWarning: An error occurred while loading the ERDDAP data fetcher, it will not be available !
```

(eg https://github.com/euroargodev/argopy-status/runs/1363053253?check_suite_focus=true)

**Comment by gmaze** [comment]
We fail to load the erddap source because: "No module named 'erddapy.utilities'"
so, we'll check if this is due to an erddap change/release

**Comment by gmaze** [comment]
and indeed: https://github.com/ioos/erddapy/releases/tag/v0.8.0
indicates: "Major refactor that removes the utilities module (#145, #148)"

**Comment by gmaze** [comment]
Fixed by commits ending in 9706739c5a4f3ac019359cc9fad487e560927af0

**Comment by gmaze** [comment]
Fixed by commits ending in 9706739c5a4f3ac019359cc9fad487e560927af0
